test/TestProjects/TypeSchemaMapping/Generated/Models/CustomizedModel.cs

```diff
@@ -12,16 +12,5 @@ namespace CustomNamespace
     /// <summary> The Model. </summary>
     internal partial class CustomizedModel
     {
-
-        /// <summary> Initializes a new instance of CustomizedModel. </summary>
-        /// <param name="propertyRenamedAndTypeChanged"> . </param>
-        /// <param name="customizedFancyField"> Fruit. </param>
-        /// <param name="daysOfWeek"> Day of week. </param>
-        internal CustomizedModel(int? propertyRenamedAndTypeChanged, CustomFruitEnum customizedFancyField, CustomDaysOfWeek daysOfWeek)
```

Comment:
I wasn't expecting this to be remove.d Isn't this the serialization ctor you need to deserialize?

Reply:
leftovers of testing